pkg/virt-handler/vm.go

```diff
@@ -177,8 +233,14 @@ func MapPersistentVolumes(vm *v1.VM, restClient cache.Getter, namespace string)
 				newDisk.Source.Protocol = "iscsi"
 
 				hostPort := strings.Split(pv.Spec.ISCSI.TargetPortal, ":")
+				ipAddrs, err := net.LookupIP(hostPort[0])
```

Comment:
I misread the code. I was under the impression that hostPort[1], which contains the port is not used anymore in this code area.